test/local/features_tests.js

```diff
@@ -30,10 +30,11 @@ test(
   'interface is correct',
   t => {
     t.equal(typeof features, 'object', 'object type should be exported')
-    t.equal(Object.keys(features).length, 3, 'object should have two properties')
+    t.equal(Object.keys(features).length, 4, 'object should have four properties')
     t.equal(typeof features.isSampledUser, 'function', 'isSampledUser should be function')
     t.equal(typeof features.isLastAccessTimeEnabledForUser, 'function', 'isLastAccessTimeEnabledForUser should be function')
     t.equal(typeof features.isSigninConfirmationEnabledForUser, 'function', 'isSigninConfirmationEnabledForUser should be function')
+    t.equal(typeof features.isSigninUnblockEnabledForUser, 'function', 'isSigninUnblockEnabledForUser should be function')
```

Comment:
Should add some tests for the the `isSigninUnblockEnabledForUser` method, specifically.